src/WebJobs.Script/Utility.cs

```diff
@@ -520,8 +520,17 @@ private static bool ContainsFunctionWithWorkerRuntime(IEnumerable<FunctionMetada
             return false;
         }
 
-        public static bool CheckAppOffline(string scriptPath)
+        public static bool CheckAppOffline(IEnvironment environment, string scriptPath)
         {
+            // Linux container environments have an additional way of putting a specific worker instance offline.
+            if (environment.IsLinuxContainerEnvironment())
```

Comment:
nit:combine these checks to avoid nesting

Reply:
done.